### PR TITLE
Add Service Worker to Site

### PR DESCRIPTION
**Pull request by agubler**
* Adds a service worker to site for all content (network first).
* Uses an appropriate icon for PWA icon

lighthouse

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,6 +10,18 @@
 	<link rel="icon" href="./assets/favicon.ico" />
 	<link rel="stylesheet" type="text/css" href="./assets/prism.css" />
 
+	<script>
```
Comment (agubler) [inline src/index.html]: Need to manually include this for the time being - as the site is static and the service worker registration is included in the js bundle...

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-gwhylgijl.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-dbtdpskiu.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-2acrbpemg.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to https://dojoio-6ampextj1.now.sh

File: content/blog/en/announcing-dojo-2-beta-4.md [hunk 2]
```diff
@@ -230,11 +230,11 @@ We have introduced two new widgets in Beta 4 and are considering more before the
 
 We have introduced a new accordion pane widget (`@dojo/widgets/accordionpane/AccordionPane`):
 
-![The accordion pane widget](/assets/blog/announcing-dojo-2-beta-4/accordion.png)
+![The accordion pane widget](assets/blog/announcing-dojo-2-beta-4/accordion.png)
```
Comment (agubler) [inline content/blog/en/announcing-dojo-2-beta-4.md]: removed the leading `/` as resources will be loaded from the application base, which by default is `/`

**Pull request merged**